app/views/admin/basic_projects/_form.html.slim

```diff
@@ -0,0 +1,46 @@
+= simple_form_for([:admin, @project],
+    html: { \
+      class: 'form-horizontal organization-record',
+    },
+    wrapper: :horizontal_form_condensed,
+    defaults: { \
+      input_html: {class: 'form-element form-control'},
+    },
+  ) do |f|
```

Comment:
At some point we need to refactor all these forms to get rid of all this boilerplate. I think I mentioned this before. This would be a really good exercise in frontend refactoring. @rgalusca can we get a form refactoring issue on the docket for next sprint? Perhaps with a 5 hour estimate? This really needs to get done and we're saving a ton of time on this issue...